.travis.yml

```diff
@@ -6,10 +6,32 @@ addons:
   hosts:
     - fake-hostname-to-work-around-travis-bug
   hostname: fake-hostname-to-work-around-travis-bug
-jdk:
-  - openjdk8
-  - openjdk7
-install: ./gradlew assemble
-script: ./gradlew build install
-after_success:
-  - bash <(curl -s https://codecov.io/bash)
+  
+jobs:
+  include:
+  - name: Java 7 Gradle Build
+    jdk: openjdk7
+    install: ./gradlew assemble
+    script: ./gradlew build install
+    after_success:
+      - bash <(curl -s https://codecov.io/bash)
+    
+  - name: Java 8 Gradle Build
+    jdk: openjdk8
+    install: ./gradlew assemble
+    script:
+      - ./gradlew build install
+    after_success:
+      - bash <(curl -s https://codecov.io/bash)
+
+  - name: Java 8 Bazel Build
+    jdk: openjdk8
+    before_install:
+      - wget https://github.com/bazelbuild/bazel/releases/download/3.5.0/bazel_3.5.0-linux-x86_64.deb
+      - wget https://github.com/bazelbuild/bazel/releases/download/3.5.0/bazel_3.5.0-linux-x86_64.deb.sha256
+      - sha256sum -c bazel_3.5.0-linux-x86_64.deb.sha256
+      - sudo dpkg -i bazel_3.5.0-linux-x86_64.deb
```

Comment:
Minor (just thought it was worth mentioning): Installation can be as easy as downloading a single binary bazel and making it executable. Maybe it is more lightweight (faster way of doing it). Downloading deb package is fine too. Maybe consider moving version of bazel in to a variable (since it is repeated here many times).

Reply:
Done.